package.json

```diff
@@ -59,7 +68,7 @@
   ],
   "husky": {
     "hooks": {
-      "pre-commit": "yarn format && yarn lint && CI=true yarn test"
```

Comment:
@tetogomez @adriexnet 

I removed this test command as it was not working properly for me